src/mito2/src/sst/index/inverted_index/creator.rs

```diff
@@ -210,6 +203,32 @@ impl InvertedIndexer {
                 .context(PushIndexValueSnafu)?;
         }
 
+        for field in batch.fields() {
+            if !self.indexed_column_ids.contains(&field.column_id) {
+                continue;
+            }
+
+            let sort_field = SortField::new(field.data.data_type());
+            let col_id_str = field.column_id.to_string();
+            for i in 0..n {
```

Comment:
I'm a little worried about the overhead for this loop 😢

Reply:
which part? `to_string` or encoding?

Reply:
> I'm a little worried about the overhead for this loop 😢

Looks like this loop lieterally iterates and builds index for each row?